proto/ibc/apps/cross-chain-validation/tx.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+package ibc.apps.ccv.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/ccv/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// MsgSubmitNewChain defines a msg to submit a new baby chain that can be validated
+// by the parent chain validators.
+message MsgSubmitNewChain {
```

Comment:
This is my proposal for the init protocol. The owner of the chain-id can create this message along with non-stake genesis file. They can specify the minimum stake they would like, and then allow validators to post their gen-txs on the parent chain.

Once minimum stake and grace period have been reached, the initial validator set is finalized. All validators can retrieve gen-txs from parent chain, collect them into intitial genesis file and start the chain.

This minimizes the need for validator communication/coordination and only requires that 2/3 of validators come online for baby chain to start (as opposed to 100%)

Will write this up as a proposal to the spec once it is in ibc repo. cc: @milosevic @josef-widder

Reply:
Sounds like gen-txs will require a separate tx message and type?